package.json

```diff
@@ -37,6 +37,7 @@
     "expose-loader": "^5.0.0",
     "jquery": "^3.7.1",
     "litemol": "https://github.com/dsehnal/LiteMol/tarball/master",
+    "marked": "^12.0.2",
```

Comment:
What's this dependency for?

Reply:
My code would not compile if I did not add it, the console was asking me to